libs/common/src/helpers/is-number-generator-mocked.ts

```diff
@@ -0,0 +1,15 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+/**
+ * checks if number generator key is mocked
+ * if numberGeneratorKey is mocked, it will have the same value as MOCK_E2E_NUMBER_GENERATOR
+ * @returns true if numberGeneratorKey is the same as MOCK_E2E_NUMBER_GENERATOR
+ */
+export const isNumberGeneratorMocked = (): boolean => {
+  const numberGeneratorKey: string = process.env.NUMBER_GENERATOR ?? '';
```

Comment:
```suggestion
  const numberGeneratorKey: string = process.env.NUMBER_GENERATOR;
```

Reply:
Dont want them to be mocked if they both are empty.
Has these been added to `.env.sample`, if not can they please be added.